tests/e2e/startup/startup_test.go

```diff
@@ -217,16 +217,16 @@ var _ = Describe("Various Startup Configurations", Ordered, func() {
 		})
 
 		It("Returns pod metrics", func() {
-			cmd := "kubectl top pod -A"
+			cmd := "kubectl top pod -A --kubeconfig=" + kubeConfigFile
 			Eventually(func() error {
-				_, err := e2e.RunCmdOnNode(cmd, serverNodeNames[0])
+				_, err := e2e.RunCommand(cmd)
 				return err
 			}, "600s", "5s").Should(Succeed())
 		})
 
 		It("Returns node metrics", func() {
-			cmd := "kubectl top node"
-			_, err := e2e.RunCmdOnNode(cmd, serverNodeNames[0])
+			cmd := "kubectl top node --kubeconfig=" + kubeConfigFile
```

Comment:
Maybe RunCmdOnNode could just consistently export this path in the KUBECONFIG environment variable so that we don't have to remember to pass it via a CLI flag all the time?